### PR TITLE
DOC-441 Added a quick start guide for generic K8s

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/platforms/kubernetes/getting-started/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+The operator currently runs within a single namespace and is scoped to operate only on the Redis Enterprise Cluster CRDs in that namespace.
+
+This lets you deploy multiple instances of the operator on the same Kubernetes cluster, supporting multiple Redis Enterprise Clusters in multiple namespaces.
+
+### Namespaces
+
+The Redis Enterprise Operator manages a single Redis Enterprise cluster in a single namespace.
+
+While the operator can watch additional namespaces for consumer namespaces, the Redis Enterprise Operator
+and cluster must exist in the same namespace.
```
Comment (banker) [inline content/platforms/kubernetes/getting-started/quick-start.md:44]: I don't understand this sentence. Can you confirm that it's correctly worded and necessary? @alexmilowski @amiramm

**Comment by banker** [comment]
@amiramm or @alexmilowski. I've finished reviewing this document. Please see my one outstanding question, and please read the doc once more, post my edits. I'm happy to merge.

**Review by banker — APPROVED** [review]

File: content/platforms/kubernetes/getting-started/quick-start.md (hunk 1 above)
Comment (amiramm) [reply content/platforms/kubernetes/getting-started/quick-start.md:43]: Let's remove it. There isn't enough context in this guide about consumer namespaces and we need to keep it simple as this should be a simple guide followed almost to the letter to get started.

File: content/platforms/kubernetes/getting-started/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+operator implementation itself is published as a Docker container. The operator
+definitions are [packaged as a single generic YAML file](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/bundle.yaml).
+
+### Download the bundle
+
+You need to ensure that you pull the correct version of the bundle. You can find the version tags
+by checking the [operator releases on GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
+or by [using the GitHub API](https://docs.github.com/en/rest/reference/repos#releases).
+
+You can download the bundle for the latest release by issuing the following `curl` commands:
```
Comment (banker) [inline content/platforms/kubernetes/getting-started/quick-start.md:66]:
```suggestion
You can download the bundle for the latest release by issuing the following `curl` command:
```

File: content/platforms/kubernetes/getting-started/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+The operator currently runs within a single namespace and is scoped to operate only on the Redis Enterprise Cluster CRDs in that namespace.
+
+This lets you deploy multiple instances of the operator on the same Kubernetes cluster, supporting multiple Redis Enterprise Clusters in multiple namespaces.
+
+### Namespaces
+
+The Redis Enterprise Operator manages a single Redis Enterprise cluster in a single namespace.
+
+While the operator can watch additional namespaces for consumer namespaces, the Redis Enterprise Operator
```
Comment (banker) [inline content/platforms/kubernetes/getting-started/quick-start.md:42]:
```suggestion
```

File: content/platforms/kubernetes/getting-started/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+and cluster must exist in the same namespace.
```
Comment (banker) [inline content/platforms/kubernetes/getting-started/quick-start.md:43]:
```suggestion
```

**Pull request merged**